metrics.go

```diff
@@ -690,7 +690,7 @@ type RPCMetrics struct {
 	OutQueue         int       `json:"outQueue"`
 	LastPongTime     time.Time `json:"lastPongTime"`
 	LastPingMS       float64   `json:"lastPingMS"`
-	Max1MinPingMS    float64   `json:"max1MinPingMS"`
+	MaxPingDurMS     float64   `json:"maxPingDurMS"` // Maximum across all merged entries.
```

Comment:
```suggestion
	MaxPingDurMS     float64   `json:"maxPingDurMS"` // Maximum across all merged RPC messages.
```

Reply:
No. It is across metric entries. Ping is not related to RPC messages - it is separate ping/pong roundtrip on the connection.